lib/daru/io/io.rb

```diff
@@ -275,6 +275,17 @@ def html_table_to_dataframe(table)
           order: table[:order],
           name: table[:name]
       end
+
+      CSV::Converters[:boolean_converter] = lambda do |f, _|
```

Comment:
Daru should convert booleans, yes. All I am agains is to modify built-in hash of `CSV::Converters`. I think it is a bad practice, generally.